src/screens/home/index.tsx

```diff
@@ -1,29 +1,109 @@
-import {Stack} from 'native-base';
-import React from 'react';
-import {Text, TouchableOpacity} from 'react-native';
+import MatchCard from 'components/shared/matchCard';
+import {Skeleton} from 'native-base';
+import React, {useCallback} from 'react';
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  FlatList,
+  SafeAreaView,
+} from 'react-native';
+import {useDispatch, useSelector} from 'react-redux';
+import {matchData, matchesToBePlayed} from 'store/modules/match/selectors';
 import {Routes, TNavigation} from 'types/interfaces';
+import {Match as IMatch} from 'store/modules/match/reducers';
+import styles from './styles';
+import {getMatches} from 'store/modules/match/actions';
+import Player5 from 'assets/svg/player5';
+import Player7 from 'assets/svg/player7';
+import {useFocusEffect} from '@react-navigation/native';
 
 const Match = ({navigation}: TNavigation<Routes.MATCH>) => {
+  const dispatch = useDispatch();
+  const matches = useSelector(matchesToBePlayed);
+  const {isLoading} = useSelector(matchData);
+
+  useFocusEffect(
+    useCallback(() => {
+      dispatch(getMatches());
+    }, [dispatch]),
+  );
```

Comment:
this will make a call to the API everytime we made focus on a screen, this is not ideal. A better approach is to call the getMatches endpoint when needed (after a successful updateMatch endpoint call)